src/LineDriver.php

```diff
@@ -67,7 +67,21 @@ public function sendRequest($endpoint, array $parameters, IncomingMessage $match
      */
     public function getUser(IncomingMessage $matchingMessage)
     {
-        return new User();
+        $userId = $this->event->get('source')['userId'];
+        
+        $userInfoData = $this->http->get($this->getApiUrl("/profile/{$userId}"), [], [
+            'Authorization: Bearer '.$this->config->get('channel_access_token'),
+            'Content-Type: application/json',
+        ], true);
+        
+        try {
+            $userInfo = json_decode($userInfoData->getContent(), true);
+            $names = self::split_name($userInfo['displayName']);
+            
+            return new User($userId, $names['first_name'], $names['last_name'], null, $userInfo);
+        } catch (\Exception $e) {
+            return new User($userId);
```

Comment:
Uh, where the exception being thrown?

And IMO, the method should handle the exceptions, instead of just returning another `User` instance.

Reply:
So I read https://github.com/botman/driver-slack/blob/master/src/SlackDriver.php and follow how they code the function. What do you think?

Reply:
https://github.com/botman/driver-slack/blob/master/src/SlackDriver.php#L386-L403

I had read the code of BotMan Facebook driver and Telegram driver but not Slack driver. It is interesting.

I got your point, but I still prefer to do things simple. If error happens, let it happen.